src/lit-html.ts

```diff
@@ -72,6 +72,10 @@ export class TemplateResult {
   }
 }
 
+export interface RenderContainer extends Element {
```

Comment:
This should be `extends ParentNode` to also pickup `DocumentFragment`.

Reply:
`extends ParentNode` does not work. `ParentNode` has no `firstChild` nor `appendChild`